gui/src/containers/Configuration/DeleteJobRuns.js

```diff
@@ -0,0 +1,116 @@
+import { withStyles } from '@material-ui/core/styles'
+import { useHooks, useState } from 'use-react-hooks'
+import { connect } from 'react-redux'
+import { bindActionCreators } from 'redux'
+import PaddedCard from 'components/PaddedCard'
+import Divider from '@material-ui/core/Divider'
+import Typography from '@material-ui/core/Typography'
+import Button from '@material-ui/core/Button'
+import Grid from '@material-ui/core/Grid'
+import Slide from '@material-ui/core/Slide'
+import {
+  deleteCompletedJobRuns,
+  deleteErroredJobRuns
+} from 'actions'
+
+const styles = theme => {
+  return {
+    confirmButton: {
+      backgroundColor: theme.palette.error.main,
+      color: theme.palette.common.white,
+      '&:hover': {
+        backgroundColor: theme.palette.error.dark
+      }
+    },
+    deleteRunsDivider: {
+      marginTop: theme.spacing.unit * 3,
+      marginBottom: theme.spacing.unit * 2
+    }
+  }
+}
+
+const WEEK_MS = (1000 * 60 * 60 * 24 * 7)
+
+const DeleteJobRuns = useHooks(({classes, deleteCompletedJobRuns, deleteErroredJobRuns}) => {
+  const updatedBefore = new Date(Date.now() - WEEK_MS).toISOString()
+  const [showCompletedConfirm, setCompletedConfirm] = useState(false)
+  const [showErroredConfirm, setErroredConfirm] = useState(false)
+
+  const confirmCompletedDelete = () => {
+    deleteCompletedJobRuns(updatedBefore)
+    setCompletedConfirm(false)
+  }
+  const confirmErroredDeleted = () => {
+    deleteErroredJobRuns(updatedBefore)
+    setErroredConfirm(false)
+  }
+
+  return (
+    <PaddedCard>
+      <Typography variant='h5' color='secondary'>
+        Delete Runs
+      </Typography>
+
+      <Typography variant='subtitle1' color='textSecondary'>
+        Reduce your database size by deleting completed and or
+        errored job runs. This action will delete runs a week ago
+        and before.
+      </Typography>
+
+      <Divider className={classes.deleteRunsDivider} />
+
+      <Grid container spacing={16}>
+        <Grid item xs={12}>
+          {!showCompletedConfirm &&
+            <Slide direction='right' in={!showCompletedConfirm} mountOnEnter unmountOnExit>
+              <Button variant='outlined' onClick={() => setCompletedConfirm(true)}>
+                Delete Completed Jobs
+              </Button>
+            </Slide>
+          }
+          {showCompletedConfirm &&
+            <Slide direction='right' in={showCompletedConfirm} mountOnEnter unmountOnExit>
+              <Button variant='contained' className={classes.confirmButton} onClick={() => confirmCompletedDelete()}>
+                Confirm delete all completed job runs up to {updatedBefore}
+              </Button>
+            </Slide>
+          }
+        </Grid>
+        <Grid item xs={12}>
+          {!showErroredConfirm &&
+            <Slide direction='right' in={!showErroredConfirm} mountOnEnter unmountOnExit>
+              <Button variant='outlined' onClick={() => setErroredConfirm(true)}>
+                Delete Errored Jobs
```

Comment:
Normalize both buttons with same widths?

Reply:
@NavyAdmiral pinged our designer.

Reply:
@rupurt 
![image](https://user-images.githubusercontent.com/40662484/49491495-63b23500-f822-11e8-9147-2d3f1f24e7a1.png)
Delete buttons have the same width in the design

Reply:
@rupurt @NavyAdmiral depending on the size of the viewport, the column will scale so we don't have total control of the width. For a stop-gap solution to delete jobs the button changing in size to confirm is not really a something we want to dump time into. 

Down the road, we chatted about a modal confirm with a more detailed description of what is going to happen with a clear confirm and cancel.